src/commons/styles/juggernaut.js

```diff
@@ -0,0 +1,25 @@
+import React, { Fragment } from 'react'
+import CssBaseline from '@material-ui/core/CssBaseline'
+import { MuiThemeProvider } from '@material-ui/core/styles'
+import { ThemeProvider } from 'styled-components'
+import GlobalStyle from './global'
+import PropTypes from 'prop-types'
+
+const Juggernaut = ({ children: Children, theme }) => (
+  <MuiThemeProvider theme={theme}>
+    <ThemeProvider theme={theme}>
+      <Fragment>
+        <CssBaseline />
+        <GlobalStyle />
+        {Children}
```

Comment:
nesse caso o padrão é manter o children em minusclo, só teria o alias para maiusculo se for executar o componente.
Porém children sempre vem executado.